src/main/scala/sparkz/core/api/http/ApiDirectives.scala

```diff
@@ -41,7 +41,7 @@ trait ApiDirectives extends CorsHandler with SparkzEncoding {
         if (apiKeyHash.equals(""))
           false
         else
-          BCrypt.checkpw(apiKey, apiKeyHash)
+          BCrypt.verifyer().verify(apiKey.toCharArray, apiKeyHash).verified
```

Comment:
let's by sure that this method doesn't throw exception.
For example:
1. apiKeyHash is dummy string
2. "apiKey.toCharArray" by sure that it is using UTF8
and so on.